infrastructure/helm-charts.tf

```diff
@@ -125,7 +125,7 @@ resource "helm_release" "custom-helm-chart" {
   }
   set{
     name = "service.port"
-    value = 80
+    value = 443
```

Comment:
on the service of type clusterIP port should still be 80, that is internal traffic between LB and K8S service